src/js/components/VoterGuide/VoterGuideBallotItemCompressed.jsx

```diff
@@ -25,8 +25,13 @@ export default class VoterGuideBallotItemCompressed extends Component {
           />
         ) : (
           <VoterGuideOfficeItemCompressed
-            {...this.props}
+            ballotItemDisplayName={this.props.ballot_item_display_name}
+            candidateList={this.props.candidate_list}
+            kindOfBallotItem={this.props.kind_of_ballot_item}
+            organization={this.props.organization}
             organizationWeVoteId={this.props.organizationWeVoteId}
+            urlWithoutHash={this.props.urlWithoutHash}
+            weVoteId={this.props.we_vote_id}
```

Comment:
Hello @MGarc758, thank you for this pull request! When I look in the file:
WebApp/src/js/components/VoterGuide/VoterGuideOfficeItemCompressed.jsx
...at:
VoterGuideOfficeItemCompressed.propTypes = {
  we_vote_id: PropTypes.string.isRequired,
  ballot_item_display_name: PropTypes.string.isRequired,
  candidate_list: PropTypes.array,
  organizationWeVoteId: PropTypes.string.isRequired,
};
...I only see 4 incoming props. Please update this code here in WebApp/src/js/components/VoterGuide/VoterGuideBallotItemCompressed.jsx to only pass in the three additional props: we_vote_id, ballot_item_display_name, and candidate_list
Thank you!

Reply:
Hi @DaleMcGrew,

I believe I added the changes requested to the VoterGuideBallotCompressed.jsx.

Thanks,
Marc